src/authBase/auth.ts

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 
 const authApi = axios.create({
-  baseURL: 'https://casa-qa.herokuapp.com/', // Ruby for Good CASA QA
+  baseURL: 'https://c9bc-142-181-45-69.ngrok-free.app', // Ruby for Good CASA QA
```

Comment:
Will change this back to QA

Reply:
actually I just had an idea
what if we used an env variable for the url? like ENV["BASE_URL_QA"]
so we don't need to change it manually every time when we test
also eventually we'll need to use env variables once we deploy to production.

Reply:
For sure, `environments` and `env's` should be another issue entirely, will note.